### PR TITLE
TandemFoil: lr=2e-4 + grad-clip=1.0 (stability test)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
Adding gradient clipping (clip=1.0) to the lr=2e-4 baseline may stabilize training and unlock further improvement. Grad-clip was transformative for AirfRANS (7.3% improvement, 91–98% of batches clipped), suggesting our Lion optimizer may also benefit from bounded gradient norms. The current best uses lr=2e-4 without clipping — this test isolates the clipping effect.

## Instructions
Run the following command exactly:

```bash
cd target/icml2026 && python train.py \
  --dataset tandemfoil \
  --optimizer lion \
  --lr 2e-4 \
  --cosine-t-max 10 \
  --grad-clip 1.0 \
  --no-use-ema \
  --model-slices 64 \
  --enable-fourier \
  --enable-te-coord-frame \
  --enable-cp-panel \
  --enable-cp-panel-tandem-only \
  --asinh-pressure \
  --residual-prediction \
  --enable-pressure-prior-addition \
  --epochs 999 \
  --wandb-group tandem-lr2e4-compounds \
  --wandb-name "naruto/tandem-lr2e4-gradclip"
```

Report the best `val_primary/surface_pressure_mae` achieved and the W&B run ID.

## Baseline
Current best metrics (PR #2610):
- val_primary/surface_pressure_mae: **45.07**
- W&B run: `ixs1rqgk`
- Epochs: 119
- Reproduce: `cd target/icml2026 && python train.py --dataset tandemfoil --optimizer lion --lr 2e-4 --cosine-t-max 10 --no-use-ema --model-slices 64 --enable-fourier --enable-te-coord-frame --enable-cp-panel --enable-cp-panel-tandem-only --asinh-pressure --residual-prediction --enable-pressure-prior-addition --epochs 999`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2728 because ADVISOR: Closing — lr=2e-4 has been superseded. The baseline is now 30.10 at lr=1.25e-4 (PR #2810). The LR descent trend 3e-4→2e-4→1.5e-4→1.25e-4 shows monotonic improvement. You'll be reassigned to a current experiment..
